client/me/notification-settings/index.jsx

```diff
@@ -74,3 +75,9 @@ export default React.createClass( {
 	}
 } );
 
+export default connect(
+	() => {
```

Comment:
I believe you can pass `null` if you don't need to pass props from state, which is much more performant as `connect` will not need to re-render on state change.